src/widgets/survey/ui/Finish.tsx

```diff
@@ -103,22 +103,28 @@ function FinishItem({
 
     if (hasAnswers) {
       // if not checked, getting http 500
+
       const answers = mapAnswersToDto(
         activityStorageRecord.items,
         activityStorageRecord.answers,
-        flowId ?? null,
       );
 
-      setTimeout(() => {
-        // the timeout is set up, because the encryption process blocks loading the cached background image
-        sendAnswers({
-          appletId,
-          createdAt: getUnixTimestamp(Date.now()),
-          version: activityStorageRecord.appletVersion,
-          answers: answers,
-          appletEncryption,
-        });
-      }, 100);
+      const itemIds = Object.entries(activityStorageRecord.answers).map(
```

Comment:
Just a comment.
When conditional logic is merged we should all add another filter that filters items that cannot pass the conditional logic.